ci_framework/playbooks/06-deploy-edpm.yml

```diff
@@ -49,10 +49,61 @@
             name: libvirt_manager
             tasks_from: deploy_edpm_compute.yml
 
+        - name: Disable discover_hosts when deploying HCI phase1
+          when: cifmw_edpm_deploy_hci | default(false) | bool
+          ansible.builtin.set_fact:
+            cifmw_edpm_deploy_skip_nova_discover_hosts: true
+
         - name: Deploy EDPM
           ansible.builtin.import_role:
             name: edpm_deploy
 
+- name: Deploy ceph deploy playbook
+  ansible.builtin.import_playbook: ceph.yml
+  when: cifmw_edpm_deploy_hci | default(false) | bool
+
+- name: Deploy remaining services when HCI environment
+  hosts: "{{ cifmw_target_host | default('localhost') }}"
+  gather_facts: false
+  tasks:
+    - name: Create Ceph secrets and retrieve FSID info
```

Comment:
The existing playbook does this for you and creates a secret in /tmp/k8s_ceph_secret.yml which you can use.

https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci_framework/playbooks/ceph.yml#L253-L270